datahub-frontend/app/auth/sso/oidc/OidcCallbackLogic.java

```diff
@@ -300,20 +304,31 @@ private CorpUserSnapshot extractUser(CorpuserUrn urn, CommonProfile profile) {
     return corpUserSnapshot;
   }
 
+  public static List<String> getGroupsClaimNames(OidcConfigs configs) {
+    List<String> groupsClaimNames;
+    String rawGroupClaimsNames = configs.getGroupsClaimName();
```

Comment:
What happens if getGroupsClaimName returns null? 

Also, isn't this just returning the _name_ of the claim, not the actual value provided in the OIDC response?

Reply:
Good call - in my refactoring I inadvertently starting using the groupsClaimName instead of the `profile.getAttribute(groupsClaimName)` . Should be fixed now